namd/tests/interface/000_stringstates/namd-version.txt

```diff
@@ -1,3 +1,3 @@
-Info: NAMD 2.14b1 for Linux-x86_64-multicore
-colvars: Initializing the collective variables module, version "2020-04-09".
-colvars: Using NAMD interface, version "2020-04-07".
+Info: NAMD 3.0b4 for Linux-x86_64-multicore
+colvars: Initializing the collective variables module, version 2023-10-03.
+colvars: Using NAMD interface, version "2023-10-03".
```

Comment:
Are the changes to this test anything more than cosmetic?

Reply:
Yes: there is a change due to a fix that removed double counting of the first timestep in repeated runs.